schemas/EGA.submission.json

```
@@ -4,10 +4,188 @@
+    "required": ["object_id"],
     "additionalProperties": false,
     "properties": {
-      
+      "object_id": {
+        "type": "object",
+        "title": "Object's IDs block",
+        "description": "Node containing the main identifiers of the object (e.g. alias, center_name...), inherited from the common definitions. #! We extend the core object (object_core_id) by adding a pattern check based on this schema's nature: a submission (by using EGA-submission-id-pattern)", 
+        "allOf": [
+          {
+            "title": "Inherited object_core_id object",
+            "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/object_core_id"
+          },
+          {
+            "title": "Check that Submission EGA ID (EGAB) is correct",
+            "properties": {
+              "ega_accession": {
+                "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/EGA-submission-id-pattern"
+              }
+            }
+          }
+        ]        
+      },
+
+      "schema_descriptor": {
+        "title": "Schema descriptor node",
+        "description": "Inherited schema descriptor node containing metadata about the schemas and standards used to create the JSON document itself.",
+        "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/schema_descriptor"
+      },
+
+      "object_title": {
+        "type": "string",
+        "title": "Title of the submission project",
+        "description": "Short free-form text that can be used to call out submission project records in searches or displays.",
+        "minLength": 1,
+        "examples": [ "Submission XF40" ]
+      },
+
+      "object_description": {
+        "type": "string",
+        "title": "Description of the submissions project",
+        "description": "An in-depth description of the submission, including its overall purpose or nature of studies it governs.",
+        "minLength": 1,
+        "examples": [ "EBI submission project XF40 of 1000 samples and its 1500 sequencing runs" ]
+      },
+
+      "additional_collaborators": {
+        "type": "array",
+        "title": "Submission ownership details",
```

```suggestion
        "title": "Submission collaborator details",
```

Based on the attribute description, I think this field is meant to capture individuals who are **not** the data owner / controller / DAC, but can still collaborate on the submission. If so, I suggest changing the attribute title.